fedmsg.d/fmn.py

```diff
@@ -23,11 +23,19 @@
     "fmn.rules.utils.use_pkgdb2": False,
     "fmn.rules.utils.pkgdb2_api_url": "http://209.132.184.188/api/",
     "fmn.rules.cache": {
-        "backend": "dogpile.cache.dbm",
-        "expiration_time": 300,
-        "arguments": {
-            "filename": "/var/tmp/fmn-cache.dbm",
+        'backend': 'dogpile.cache.redis',
+        'arguments': {
+            'host': 'localhost',
+            'port': 6379,
+            'db': 0,
+            'redis_expiration_time': 60*60*2,   # 2 hours
+            'distributed_lock': True
         },
+        # "backend": "dogpile.cache.dbm",
+        # "expiration_time": 300,
+        # "arguments": {
+        #     "filename": "/var/tmp/fmn-cache.dbm",
+        # },
```

Comment:
I guess since we are forcing sse we should, but what if we have an instance we don't want to host sse? then dogpile cache would work